Cannot publish to OpenVSX due to out of date NodeJS

When attempting to publish to Open VSX via GitHub Actions:

```
Run ovsx publish -p ***  --packagePath vscode-quarkus/vscode-quarkus-*-${GITHUB_RUN_NUMBER}.vsix
ovsx requires at least NodeJS version 20. Check your installed version with `node --version`.
Error: Process completed with exit code 1.
```

We should upgrade the version of node used in the CD pipeline, since node 18 is no longer supported.

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -92,7 +92,7 @@ jobs:
       - name: Set Up NodeJS
         uses: actions/setup-node@v4
         with:
-          node-version: '18'
+          node-version: '20'
       - name: Install NodeJS dependencies
         run: npm install -g typescript "@vscode/vsce" "ovsx"
       - name: Build Extension
@@ -142,7 +142,7 @@ jobs:
       - name: Set Up NodeJS
         uses: actions/setup-node@v4
         with:
-          node-version: '18'
+          node-version: '20'
       - name: Install dependencies
         run: |
           npm install -g typescript "@vscode/vsce" "ovsx"
@@ -166,7 +166,7 @@ jobs:
       - name: Set Up NodeJS
         uses: actions/setup-node@v4
         with:
-          node-version: '18'
+          node-version: '20'
       - name: Upversion for Development
         run: |
           tag=`npm version --no-git-tag-version patch`

